### PR TITLE
fix typos and obvious errors

### PR DESCRIPTION
**Pull request by sorawee**
Note that I fix typos only in English lang. Typos in Spanish lang are only fixed when I "search & replace all" in English lang.

### CODE REVIEW

File: courses/algebra/units/langs/english/unit9/the-unit.scrbl [hunk 1]
```diff
@@ -82,7 +82,7 @@
                            @item{Student @(resource-link #:path "workbook/StudentWorkbook.pdf" #:label "workbook")}
                            @item{Pens/pencils for the students, fresh whiteboard markers for teachers}
                            @item{Class posters (List of rules, basic skills, course calendar)}
-                           @item{Language Table (seeß below)}
```
Comment (sorawee) [inline courses/algebra/units/langs/english/unit9/the-unit.scrbl]: Not sure what to do with this...

File: courses/reactive/units/langs/english/unit5/the-unit.scrbl [hunk 2]
```diff
@@ -214,7 +214,7 @@ one who acts as the reactor, and one who acts as the @code{draw-state} function.
 each student a few sheets of paper and something to write with.
 
 @itemlist[
-@item{When a key is "pressed" by the keyboard, the reactor write doesn the current
```
Comment (sorawee) [inline courses/reactive/units/langs/english/unit5/the-unit.scrbl]: Not exactly sure what this is supposed to mean...

**Comment by schanzer** [comment]
Paging @Emmay

File: courses/data-science/units/langs/english/unit1/the-unit.scrbl [hunk 3]
```diff
@@ -385,7 +385,7 @@
                     @student{
                             The @code{triangle} function needs a Number and two Strings as input, and it produces an Image. As you can imagine, there are many other functions for making images, each with a different set of arguments. For each function we learn, we need to keep track of three things:
                             @itemlist[
-                                @item{@bold{Name} - the name of the function, which we type in whenver we want to use it}
```
Comment (sorawee) [inline courses/data-science/units/langs/english/unit1/the-unit.scrbl]: This is still wrong, @schanzer.

File: courses/data-science/units/langs/english/unit3/the-unit.scrbl [hunk 4]
```diff
@@ -249,7 +249,7 @@
                 }
                 @point{
                       @student{
-                              @activity[#:forevidence "BS-IDE&1&1" "Data 3.1.1&1&1" "Data 3.1.1&1&2" "Data 3.1.1&1&4"]{
```
Comment (sorawee) [inline courses/data-science/units/langs/english/unit3/the-unit.scrbl]: @schanzer Ditto.

File: courses/data-science/units/langs/english/unit3/the-unit.scrbl [hunk 5]
```diff
@@ -291,7 +291,7 @@
                 }
                 @point{
                       @student{
-                              @activity[#:forevidence "BS-IDE&1&1" "Data 3.1.1&1&3" "Data 3.1.1&1&4"]{
```
Comment (sorawee) [inline courses/data-science/units/langs/english/unit3/the-unit.scrbl]: @schanzer Ditto.

File: courses/data-science/units/langs/english/unit4/the-unit.scrbl [hunk 6]
```diff
@@ -18,7 +18,7 @@
                                               @code{.row-n, .order-by, .filter, .build-column}
                                               ""))]{
   @unit-descr{
-      Students construct Bar and Pie Charts in Pyret, to visualize the @italic{quantities} present in rows of their dataset. They experiment with these visualizations in a contrived dataset, apply them to their own research, and intepret the results. They also begin to write their own Sample Tables
```
Comment (sorawee) [inline courses/data-science/units/langs/english/unit4/the-unit.scrbl]: @schanzer Ditto.

File: courses/data-science/units/langs/english/unit6/the-unit.scrbl [hunk 7]
```diff
@@ -18,7 +18,7 @@
                                               @code{.row-n, .order-by, .filter, .build-column, bar-chart, pie-chart, mean, median, modes} 
                                               ""))]{
   @unit-descr{
-    Students explore new visualizations in Pyret, this time focusing on the @italic{frequency} of observations in their dataset. They learn how to construct and interpret Frequency Bar Charts and Histograms, experiment with these visualizations in a contrived dataset, apply them to their own research, and intepret the results.
```
Comment (sorawee) [inline courses/data-science/units/langs/english/unit6/the-unit.scrbl]: @schanzer Ditto.

File: courses/data-science/units/langs/english/unit8/the-unit.scrbl [hunk 8]
```diff
@@ -132,8 +132,8 @@
 
                                 @code[#:multi-line #t]{
                                         # use linear regression to extract a predictor function
-                                        lr-plot :: (t :: Table, xs :: Str, ys :: Str) -> Image
```
Comment (sorawee) [inline courses/data-science/units/langs/english/unit8/the-unit.scrbl]: @schanzer Ditto.

File: courses/data-science/units/langs/english/unit4/the-unit.scrbl (hunk 6 above)
Comment (schanzer) [reply courses/data-science/units/langs/english/unit4/the-unit.scrbl]: What's wrong here?

File: courses/data-science/units/langs/english/unit6/the-unit.scrbl (hunk 7 above)
Comment (schanzer) [reply courses/data-science/units/langs/english/unit6/the-unit.scrbl]: ?

File: courses/data-science/units/langs/english/unit8/the-unit.scrbl (hunk 8 above)
Comment (schanzer) [reply courses/data-science/units/langs/english/unit8/the-unit.scrbl]: ?

File: courses/algebra/units/langs/english/unit9/the-unit.scrbl (hunk 1 above)
Comment (schanzer) [reply courses/algebra/units/langs/english/unit9/the-unit.scrbl]: Nothing needs to be done here.

File: courses/data-science/units/langs/english/unit8/the-unit.scrbl (hunk 8 above)
Comment (sorawee) [reply courses/data-science/units/langs/english/unit8/the-unit.scrbl]: `Str` is not a valid annotation. `String` is.

Comment (schanzer) [reply courses/data-science/units/langs/english/unit8/the-unit.scrbl]: This is not a mistake - Str is an intentional abbreviation for String

**Comment by schanzer** [comment]
THANKS, @sorawee !!!

**Pull request merged**